esp/bindings/SOAP/client/soapclient.hpp

```diff
@@ -45,6 +49,20 @@ class esp_http_decl CSoapClient : implements ISoapClient, public CInterface
     CSoapClient(){m_disableKeepAlive=false;}
     CSoapClient(ITransportClient* transportclient){m_transportclient.setown(transportclient); m_disableKeepAlive=false;}
     virtual ~CSoapClient() {};
+
+    void setConnectTimeoutMs(unsigned val)
+    {
+        connectTimeoutMs = val;
+        if (m_transportclient)
+            static_cast<IHttpClient*>(m_transportclient.get())->setConnectTimeOutMs(connectTimeoutMs); //until we support something other than soap over http, static_cast
```

Comment:
I'm a little conflicted... Makes me wonder if the connection options should be exposed in the transportclient interface... 
I'm not sure if these particular timeouts are http specific, and don't belong in the transport layer...